### DISCUSSION TITLE
BBob with Vue3

### DISCUSSION THREAD

**Discussion by lukasvdgaag** [JiLiZART/BBob#169] [Q&A] [answered]
Good afternoon,

I was wondering if there were any plans to add Vue3 support for this package. It looks really promising but unfortunately, the Vue2 version does not work on Vue3, as expected.

Would love to use it.
If there is a way to use the current versions with Vue3 and I missed it, please let me know ☺️

**Comment by JiLiZART** [comment, marked as answer]
Yes, I have plans to add support of Vue3
